website_blog_tags_variable_dimension/views/templates.xml

```diff
@@ -24,16 +23,13 @@
             <xpath expr="//div[@id='blog_right_column']/section[1]" position="before">
                 <section class="mt32">
                     <h4>Tags</h4>
-                    <ul class="nav nav-pills nav-stacked">
+                    <ul class="nav nav-pills nav-stacked blog-tagcloud">
                         <t t-set="random_tags" t-value="tags.randomize_recordset()"/>
                         <t t-foreach="random_tags" t-as="nav_tag">
-                            <li t-att-class="tag and tag.id == nav_tag.id and 'active' or None" 
-                                t-attf-style="display: inline-block; font-size: {{nav_tag.font_size_in_cloud}}px;">
+                            <li t-att-class="'active' if tag and tag.id == nav_tag.id else None" 
+                                t-attf-style="font-size: {{nav_tag.font_size_in_cloud}}px;">
                                 <a t-attf-href="#{blog_url(tag=nav_tag)}">
-                                    <span t-attf-style="color:{{nav_tag.random_color}};" t-field="nav_tag.name"/>
-                                    <span t-attf-style="color:{{nav_tag.random_color}};"> (</span>
-                                    <span t-attf-style="color:{{nav_tag.random_color}};" t-esc="nav_tag.tag_cardinality"/> 
-                                    <span t-attf-style="color:{{nav_tag.random_color}};"> )</span>
+                                    <span t-attf-style="color:{{nav_tag.random_color}};"><t t-esc="nav_tag.name"/> (<t t-esc="nav_tag.tag_cardinality"/>)</span>
```

Comment:
Looks nice , thank  you, ready  to merge today. 
Could you come up with an elegant idea for the vertical/horizontal mixed cloud?   the typical variable dimension "crossword puzzle"  cloud?